<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>souvenirShop</class>
 <widget class="QDialog" name="souvenirShop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="souvCart_tableView">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="souv_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Souvenirs form: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="souv_comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantity_spinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="doneBuying">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DONE</string>
   </property>
  </widget>
  <widget class="QTableView" name="souv_tableView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="selectCampus_comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addSouvenir_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="souvCart_label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Souvenir Cart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainPagrButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Main Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="finalCostLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
